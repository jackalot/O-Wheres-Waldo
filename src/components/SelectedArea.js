import React from "react";
import "../css-components/SelectedArea.css";
function SelectedArea(props) {
  return (
    <div
      style={{
        position: "absolute",
        left: `${props.xPosition - 20}px`,
        top: `${props.yPosition - 20}px`,
      }}
      className="selectedArea-container"
    ></div>
  );
}

export default SelectedArea;
