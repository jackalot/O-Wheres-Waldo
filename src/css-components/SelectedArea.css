.selectedArea-InnerContainer {
   height: 80px;
   width: 80px;
   border-style: dashed;
  border-radius: 50%;
}