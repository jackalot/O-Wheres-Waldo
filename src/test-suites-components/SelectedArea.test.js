import { getByRole, render, screen } from "@testing-library/react";
import SelectedArea from "../components/SelectedArea";

describe("SelectedArea component", () => {
  test("Selected Area div is rendered", () => {
    const { container } = render(<SelectedArea></SelectedArea>);
    expect(container).toMatchSnapshot();
  });
});
