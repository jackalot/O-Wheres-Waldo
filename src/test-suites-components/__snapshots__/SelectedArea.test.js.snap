// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedArea component Selected Area div is rendered 1`] = `
<div>
  <div
    class="selectedArea-OuterContainer"
  >
    <div
      class="selectedArea-InnerContainer"
      style="position: absolute;"
    />
  </div>
</div>
`;
